qa/tasks/cephfs/test_admin.py

```diff
@@ -122,6 +122,28 @@ def wait_till_health_warn(self, health_warn, active_mds_id, sleep=3,
                 if health_warn in health_report['checks']:
                     return
 
+    def gen_health_warn_mds_cache_oversized(self):
```

Comment:
Commit efdb4a9ef10295b1871d9bc067a128a2eef5f059 says "move them to a better location" but these helpers are in test_admin.py source. Also, now that these are generic helpers, IMO there are places where these can be reused, right?

Reply:
I think the same. Therefore they were moved to CephFSTestCase originally . But @batrick recommended to keep them in test_admin.py since these are specific to nuances of the test. See https://github.com/ceph/ceph/pull/57333#discussion_r1594182820.

IMO, if these are moved to CephFSTestCase they can be reused to generate same warnings in future by other tests and some other tests, like test_client_limits.py, can re-use them instead of having it's own helper to generate same warning.

Reply:
I don't mind it living in test_admin.py, but then, fix the commit message please.

Reply:
Copying commit message here for reference -
```
    qa/cephfs: make code for generating health warnings reusable
    
    Code to generate MDS_TRIM and MDS_CACHE_OVERSIZED health warnings is
    repeated in test methods of TestMDSFail and TestFSFail. Move this code
    to separate helper methods so that it can be reused instead of
    duplicating it. And move these helper methods to TestAdminCommands so
    to make them conveniently available for reuse.

```

I think you dislike that it says "And move them to a better location for easier reuse in future." By that I meant we are moving it from TestFSFail and TestMDSFail to TestAdminCommands so that it can be used by other tests in test_admin.py. I've updated it accordingly. Let me know if that's okay.

Reply:
@vshankar ping

Reply:
Will be moving this commit to separate PR since Patrick requested it.